src/lsp/cobol_parser/keywords/gen_keywords.ml

```diff
@@ -78,10 +82,13 @@ let pp_terminal ppf t =
 let cobolize name =
   String.map (function '_' -> '-' | c -> c) name
 
+let combined_cobolize name =
```

Comment:
`combined` will suffice.

Reply:
```ocaml
let combined attrs =
  List.find_opt (Attribute.has_label "keyword.combined") attrs |>
  Option.map Attribute.payload
```
There is already a function with that name :upside_down_face:

Reply:
I see. `combined_name` maybe (replacing underscores is not really a "cobolization" in the sense of `cobolize` above).